CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## Unreleased
+
+### Added
+
+- The API `PATCH` methods for `machine-config` can now be used to reset
+  the `cpu_template` to `"None"`. Until this change there was no way to
+  reset the `cpu_template` once it was set.
+
+### Changed
+
+- The API `PATCH` method for `/machine-config` can be now used to change
+  `track_dirty_pages` on aarch64.
+
+### Fixed
+
+- Fixed inconsistency that allowed the start of a microVM from a JSON file
+  without specifying the `vcpu_count` and `mem_size_mib` parameters for
+  `machine-config` although they are mandatory when configuring via the API.
+  Now these fields are mandatory when specifying `machine-config` in the JSON
+  file and when using the `PUT` request on `/machine-config`.
+- Fixed inconsistency that allowed a user to specify the `cpu_template`
+  parameter and set `smt` to `True` in `machine-config` when starting from a
+  JSON file on aarch64 even though they are not permitted when using `PUT` or
+  `PATCH` in the API. Now Firecracker will return an error if `smt` is set to
+  `True` or if `cpu_template` is specified on aarch64.
```

Comment:
nit: 
```suggestion
  `True` on aarch64 or if `cpu_template` is specified on aarch64.
```

It's repetitive but I think it's more clear

Reply:
Ok, rephrased.